parsl/executors/taskvine/factory_config.py

```diff
@@ -0,0 +1,120 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+# Configuration of a TaskVine factory
```

Comment:
Maybe this should be the first sentence of the docstring so that it appears as the short description of the structure in docs? (and same for TaskVineManagerConfig)

Reply:
Yes, corrected.